utils/carCard.js

```diff
@@ -0,0 +1,38 @@
+import cars from '../data/cars.js';
+
+const generateCarCard = () => {
+  const carsSection = document.querySelector('.cars');
```

Comment:
- [optional] Great job with the mobile menu. Please ensure that when the menu pops up, it should not be possible for a user to scroll through the app
- [optional] it will be nice if you add a bit of background to beatify the mobile popup menu. Great job
![Screenshot (451)_LI](https://user-images.githubusercontent.com/65298048/190678930-fd5c7846-be55-4d60-9bca-f6b1a47acf9c.jpg)